src/App.tsx

```diff
@@ -575,9 +588,17 @@ function App(): ReactElement {
             Copy URL
           </Button>
 
-          <Button type="primary" disabled={true}>
-            Export
-          </Button>
+          <ExportButton
+            totalFrames={dataset?.numberOfFrames || 0}
+            setFrame={setFrame}
+            currentFrame={currentFrame}
+            startRecording={(options: Partial<RecordingOptions>) => {
+              recordingControls.start(setFrameAndRender, getCanvasImageAsUrl, options);
+            }}
+            stopRecording={() => recordingControls.abort()}
+            defaultImagePrefix={datasetKey + "-" + featureName + "-"}
+            disabled={dataset === null}
+          />
```

Comment:
App.tsx acts as a bridge between RecordingControls and the export button here. Both are mostly agnostic to the other besides shared parameter typing.